spec/jobs/mentions/create_all_job_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Mentions::CreateAllJob do
+  let(:comment) { create(:comment, commentable: create(:article)) }
+
+  describe ".perform_later" do
```

Comment:
This test is fine, but you can shorten testing `perform_later` by using shared examples `"#enqueues_job"`